media-gfx/gnofract4d/gnofract4d-4.3_p20230717.ebuild

```diff
@@ -46,11 +46,16 @@ src_prepare() {
 }
 
 python_test() {
+	ln -s "${BUILD_DIR}"/lib/fract4d/*.so fract4d/ || die
```

Comment:
Remind me again, did this leave the .so file present which was a problem? If yes, maybe it can always be linked and removed before `pkg_*inst` phases.